my-page-app/src/components/newemployee/NewEmployeeForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ChangeEvent } from 'react'
+import styles from './NewEmployeeForm.module.scss'
+import { NewEmployeeDTO } from '@/types'
+
+
+type Props = {
+  inputData: NewEmployeeDTO,
+  setInputData: React.Dispatch<React.SetStateAction<NewEmployeeDTO>>
```

Comment:
Can remove `React` here.
```suggestion
  setInputData: Dispatch<SetStateAction<NewEmployeeDTO>>
```